diff --git a/dist/CMakeLists.txt b/dist/CMakeLists.txt
index 3b4cb45b..cbca1ca0 100644
--- a/dist/CMakeLists.txt
+++ b/dist/CMakeLists.txt
@@ -1,6 +1,6 @@
 
 if(UNIX)
 	install(FILES stuntrally.desktop sr-editor.desktop DESTINATION "share/applications/")
-	install(FILES ../data/gui/stuntrally.png ../data/gui/sr-editor.png DESTINATION "share/icons/hicolor/64x64/apps/")
+	install(FILES ../data/gui/sr-editor.png DESTINATION "share/icons/hicolor/64x64/apps/")
 endif()
 
