diff --git a/Makefile.am b/Makefile.am
index c54b2cc..3863954 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -3,8 +3,8 @@ ACLOCAL_AMFLAGS = -I m4
 WANT_AUTOMAKE = 1.7
 WANT_AUTOCONF = 2.57
 
-DIST_SUBDIRS = zzip zzipwrap bins test docs  SDL
-SUBDIRS =      zzip zzipwrap bins test docs @SDL@
+DIST_SUBDIRS = zzip zzipwrap bins test SDL
+SUBDIRS =      zzip zzipwrap bins test @SDL@
 
 rpm: dist-bzip2 $(PACKAGE).spec
 	rpmbuild -ta $(PACKAGE)-$(VERSION).tar.bz2
@@ -62,12 +62,6 @@ osc-distclean:
 indent-check:
 	(cd zzip && $(MAKE) `basename $@`)
 
-doc docs docu clean-doc clean-docs clean-docu   zzip.html zzip.xml zzip.pdf \
-man mans manpages htmpages unpack clean-unpack  changes.htm pdfs \
-omf install-omf install-doc install-docs install-man3 install-mans \
-upload-sourceforge www:
-	(cd docs && $(MAKE) `basename $@`)
-
 sdl testsdl test-sdl install-sdl :
 	if test -z "$(SDL)"; then echo "skip $@ (no SDL)"; else \
 	(cd SDL && $(MAKE) `basename $@`) ; fi
@@ -104,7 +98,7 @@ msvc6/zziptest.dsp     msvc6/zzip.dsp
 EXTRA_DIST =   zziplib.spec zzipback.sed Makefile.mk \
                $(MSVC8) $(MSVC7) $(MSVC6) $(am__aclocal_m4_deps)
 
-PHONY = auto boottrap rpm doc docs man manpages htmpages sdl testsdl \
+PHONY = auto boottrap rpm man manpages htmpages sdl testsdl \
 	comp compats msvc6 msvc7
 # ------------------------------------------------------------------------
 
diff --git a/Makefile.in b/Makefile.in
index f8b38ff..959091d 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -392,8 +392,8 @@ AUTOMAKE_OPTIONS = 1.4 foreign dist-bzip2
 ACLOCAL_AMFLAGS = -I m4
 WANT_AUTOMAKE = 1.7
 WANT_AUTOCONF = 2.57
-DIST_SUBDIRS = zzip zzipwrap bins test docs  SDL
-SUBDIRS = zzip zzipwrap bins test docs @SDL@
+DIST_SUBDIRS = zzip zzipwrap bins test SDL
+SUBDIRS = zzip zzipwrap bins test @SDL@
 OSC_ROOT = /my/own/osc
 OSC_SRCDIR = $(OSC_ROOT)
 OSC_PROJECT = home:guidod:zziplib-13
@@ -431,7 +431,7 @@ msvc6/zziptest.dsp     msvc6/zzip.dsp
 EXTRA_DIST = zziplib.spec zzipback.sed Makefile.mk \
                $(MSVC8) $(MSVC7) $(MSVC6) $(am__aclocal_m4_deps)
 
-PHONY = auto boottrap rpm doc docs man manpages htmpages sdl testsdl \
+PHONY = auto boottrap rpm man manpages htmpages sdl testsdl \
 	comp compats msvc6 msvc7
 
 
@@ -961,12 +961,6 @@ osc-distclean:
 indent-check:
 	(cd zzip && $(MAKE) `basename $@`)
 
-doc docs docu clean-doc clean-docs clean-docu   zzip.html zzip.xml zzip.pdf \
-man mans manpages htmpages unpack clean-unpack  changes.htm pdfs \
-omf install-omf install-doc install-docs install-man3 install-mans \
-upload-sourceforge www:
-	(cd docs && $(MAKE) `basename $@`)
-
 sdl testsdl test-sdl install-sdl :
 	if test -z "$(SDL)"; then echo "skip $@ (no SDL)"; else \
 	(cd SDL && $(MAKE) `basename $@`) ; fi
diff --git a/configure b/configure
index f1306de..eb4162a 100755
--- a/configure
+++ b/configure
@@ -16619,7 +16619,7 @@ aclocaldir="$ax_warning_default_aclocal_dir"
 
 
 
-ac_config_files="$ac_config_files Makefile zzip/Makefile zzipwrap/Makefile SDL/Makefile docs/Makefile test/Makefile bins/Makefile"
+ac_config_files="$ac_config_files Makefile zzip/Makefile zzipwrap/Makefile SDL/Makefile test/Makefile bins/Makefile"
 
 cat >confcache <<\_ACEOF
 # This file is a shell script that caches the results of configure
@@ -17679,7 +17679,6 @@ do
     "zzip/Makefile") CONFIG_FILES="$CONFIG_FILES zzip/Makefile" ;;
     "zzipwrap/Makefile") CONFIG_FILES="$CONFIG_FILES zzipwrap/Makefile" ;;
     "SDL/Makefile") CONFIG_FILES="$CONFIG_FILES SDL/Makefile" ;;
-    "docs/Makefile") CONFIG_FILES="$CONFIG_FILES docs/Makefile" ;;
     "test/Makefile") CONFIG_FILES="$CONFIG_FILES test/Makefile" ;;
     "bins/Makefile") CONFIG_FILES="$CONFIG_FILES bins/Makefile" ;;
 
diff --git a/configure.ac b/configure.ac
index 9d45f34..c5ec620 100644
--- a/configure.ac
+++ b/configure.ac
@@ -373,7 +373,7 @@ AX_ENABLE_DEFAULT_ACLOCALDIR
 AC_SUBST([CONFIG_FILES])
 AC_OUTPUT([Makefile
            zzip/Makefile zzipwrap/Makefile SDL/Makefile
-           docs/Makefile test/Makefile bins/Makefile])
+           test/Makefile bins/Makefile])
 dnl -------------------------------------------------------------
 PATCH_LIBTOOL_ON_DARWIN_ZSH_OVERQUOTING
 PATCH_LIBTOOL_SYS_LIB_SEARCH_PATH_SPEC
